site/src/pages/design/actionable-language.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: Actionable Language
+category: Content
+---
+
+# Actionable Language
```

Comment:
Nice. I think this could generally use a redesign at some point to make it a little more intuitive where to find the usage guides vs pattern guides but this is a nice flow for now

Reply:
Yeah, wasn't really sure where to put this stuff, but i think it works